src/main/java/com/minecolonies/core/compatibility/jei/JobBasedRecipeCategory.java

```diff
@@ -285,7 +286,8 @@ public InfoBlock(final String text, final String tip, final Rect2i bounds)
     @NotNull
     private static EntityCitizen createCitizenWithJob(@NotNull final IJob<?> job)
     {
-        final EntityCitizen citizen = new EntityCitizen(ModEntities.CITIZEN, FAKE_LEVEL);
+        final FakeLevel level = new SingleBlockFakeLevel(Minecraft.getInstance().level);
```

Comment:
couldn't we "static cache" this?

Reply:
We could, and probably should, but I was going to do that in a separate PR because 1.20 should do something similar.

Reply:
kk